ios/PresentationLayer/SampleComposeMultiplatform/Sources/SampleComposeMultiplatform/SampleComposeMultiplatformView.swift

```diff
@@ -19,17 +19,22 @@ struct SampleComposeMultiplatformView: View {
         self.flowController = flowController
     }
     
-    var body: some View {
-        ComposeViewController {
-            SampleComposeMultiplatformScreenViewControllerKt.SampleComposeMultiplatformScreenViewController(
-                onEvent: { event in
-                    switch onEnum(of: event) {
-                    case .showMessage(let message):
-                        toastData = ToastData(message.message, hideAfter: 2)
+    public var body: some View {
+        ZStack {
+            ComposeViewController {
+                SampleComposeMultiplatformScreenViewControllerKt.SampleComposeMultiplatformScreenViewController(
```

Comment:
Prefix no longer necessary?

Reply:
Nope, thanks to Skie ❤️

Reply:
Can we delete it then? 🙏

Reply:
Ohhh, I'm so sorry, I thought it was already removed in the MR 😅 sure, I removed it 🙂